src/lib/core/CHIPConfig.h

```diff
@@ -803,10 +819,22 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
  * The default value comes from 3path per subsctipion * 3sub per fabric * max number of fabrics, then reserve 1 read client with 9
  * paths for each fabric.
  */
-#ifndef CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS
-// #define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS (CHIP_CONFIG_MAX_FABRICS * 18)
-// TODO: (#17085) Should be 3 sub * 3 path + 9 path (for read) = 18
-#define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS (CHIP_CONFIG_MAX_FABRICS * 13)
+#ifndef CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS_FOR_SUBSCRIPTIONS
+// #define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS (CHIP_IM_MAX_NUM_SUBSCRIPTIONS * 3)
+// TODO: (#17085) Should be (CHIP_IM_MAX_NUM_SUBSCRIPTIONS * 3)
+#define CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS_FOR_SUBSCRIPTIONS (CHIP_IM_MAX_NUM_SUBSCRIPTIONS * 2)
```

Comment:
Please fix the documentation above, which has the wrong `@def` and the wrong `@brief` now.

Reply:
Fixed, found the "this value comes from ..." is not necessary since the current expression is easy enough to understand.